src/CoreBundle/Entity/UserRelUser.php

```diff
@@ -27,13 +27,35 @@ class UserRelUser
      */
     protected $id;
 
+//    /**
+//     * @var int
+//     *
+//     * @ORM\Column(name="user_id", type="integer", nullable=false)
+//     */
+//    protected $userId;
     /**
-     * @var int
+     * @ORM\OneToOne (targetEntity="Chamilo\CoreBundle\Entity\User",
+     *      inversedBy="user_rel_user")
```

Comment:
`user_rel_user` is incorrect for `inversedBy`. Because it should be the class property in User entity.